src/h3/api/_api_template.py

```diff
@@ -17,22 +17,43 @@ def _api_functions(
         _globals,
 ):
     def versions():
-        return c.versions()
+        """ Return version numbers for both the Python library
+        and the underlying H3 C library.
+
+        The two versions should match exactly.
+
+        Returns
+        -------
+        Dict[str, str]
+            Consists of keys `'c'` and `'python'`.
+        """
+        return _a.versions()
 
     def string_to_h3(h):
-        return c.hex2int(h)
+        """ Converts a hexadecimal string to an H3 64-bit integer index
+        """
+        return _a.hex2int(h)
 
     def h3_to_string(x):
-        return c.int2hex(x)
+        """ Converts an H3 64-bit integer index to a hexadecimal string
+        """
+        return _a.int2hex(x)
 
     def num_hexagons(resolution):
-        return c.num_hexagons(resolution)
+        """ Return the total number of *cells* (hexagons and pentagons)
```

Comment:
That's a good point - this should be `numIndexes` in the core lib in v4.

Reply:
What about `numCells`? An "index" could still be a unidirectional edge, right?

Reply:
I think there's some general terminology we ought to work out for v4... currently we've been using `index` consistently for cell index, but it might be worth refining.